tests/test_gptrainer.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+from sklearn.metrics import roc_auc_score
+
+import modelgym
+from modelgym.gp_trainer import GPTrainer
+from modelgym.util import TASK_CLASSIFICATION, split_and_preprocess
+
+TEST_SIZE = 0.2
+N_CV_SPLITS = 2
+N_PROBES = 10
+N_ESTIMATORS = 100
+PARAMS_TO_TEST = ['eval_time', 'status', 'params']
+FIT_TEST = (PARAMS_TO_TEST[:])
+FIT_TEST.extend(['bst', 'n_estimators'])
+CV_FIT = (PARAMS_TO_TEST[:])
+CV_FIT.extend(['gp_eval_num', 'best_n_estimators'])
+MAX_ROC_AUC_SCORE = 1.0
+
+
+@pytest.mark.usefixtures("preprocess_data")
+def test_crossval_fit_eval(preprocess_data):
```

Comment:
isn't it looking exactly like test of a regular trainer? don't we need to parametrize it like test_fit for models?